components/DepartmentsWalkthrough.vue

```diff
@@ -0,0 +1,297 @@
+<template>
+  <v-container class="Category-Background carousel-view" fluid>
+    <transition-group class="carousel" tag="div">
+      <div
+        v-for="(department, i) in departments"
+        v-bind:key="department.department"
+        v-bind:style="{display: department.display}"
+        class="slide"
+      >
+        <v-row>
+          <v-col class="Category-Title">{{i+1}}. {{department.header}}</v-col>
+        </v-row>
+        <v-row class="Category-Content">
+          <v-col cols="7">
+            <v-row class="Category-Content-Text text-sm-body-1">
+              <span v-html="department.content"></span>
+            </v-row>
+            <v-row justify="center">
+              <v-btn id="btn" elevation="0" outlined color="#BDBDBD" @click="expand">SEE MORE v</v-btn>
+            </v-row>
+          </v-col>
+          <v-col cols="1"></v-col>
+          <v-col>
+            <v-row>
+              <div class="Category-Slider">
+                <div class="Slider-Amount">${{department.storedValue}} mil</div>
+                <v-row justify="center">
+                  <v-slider
+                    v-bind:max="department.sliderMax"
+                    v-model="department.storedValue"
+                    min="0"
+                    vertical
+                    track-color="#B6DADA"
+                    color="#2A6465"
+                    @mousedown="tradeBtns"
+                  />
+                </v-row>
+              </div>
+            </v-row>
+            <v-row
+              class="Category-Budget-Caption text-sm-body-1"
+            >Last year’s budget: {{department.budget}}</v-row>
+          </v-col>
+        </v-row>
+      </div>
+    </transition-group>
+    <div class="carousel-controls">
+      <v-btn
+        v-for="(button) in buttonsShow"
+        v-bind:key="button.name"
+        class="carousel-controls__button"
+        v-bind:class="button.class"
+        v-bind:disabled="button.disabled"
```

Comment:
I think the goal of the `tradeBtns` is to toggle the "Next" button to be disabled or not, but there's probably a nicer way to do this, e.g. binding the button's `disabled` attribute to a method that checks if the department has a non-zero `storedValue`. Something like this:
```suggestion
        v-bind:disabled="department.storedValue"
```

Or maybe you'd have to have an `activeDepartment` data field and use `activeDepartment.storedValue`, idk. thoughts?

Reply:
Then we can get rid of `tradeBtns` and the `next-disabled` button, since rendering buttons with a `v-for` rather than conditional `v-if`s seems weird to me, unless it's idiomatic in Vue

Reply:
i think this is a good suggestion! i tried to implement it quickly but had issues that i couldn't overcome quickly. i tried to implement the following after updating the departments loop to expand to the buttons:
`v-bind:disabled="(department.storedValue == 0) & (button.text == 'next')"`
but for some reason it caused the navigate function not to work altogether. for now both buttons will be disabled until the slider is moved but we can update that down the road.

Reply:
also i had errors when using `v-for` and `v-if` together -- documentation online says that it's allowed but not best practice but i still got errors.